src/database/models.py

```diff
@@ -259,25 +260,40 @@ class Media(models.Model):
     is_deleted = models.BooleanField(default=False, blank=True)
     order = models.PositiveIntegerField(default=0, blank=True, null=True)
     tags = models.ManyToManyField(Tag, related_name="media_tags", blank=True)
+    more_info = models.JSONField(blank=True, null=True)
```

Comment:
The description says "No DB changes".  This does require a migration.

Reply:
You can ignore that change. It was for testing in initial stages when I was playing with the django interface. (Ignoring it or making the db change wont break anything)

Reply:
Then please take it out.